addons/sys_attenuate/XEH_preInit.sqf

```diff
@@ -1,7 +1,5 @@
 #include "script_component.hpp"
 
-NO_DEDICATED;
-
 ADDON = false;
 
 PREP_RECOMPILE_START;
```

Comment:
`GVAR(attenuationCache) = HASH_CREATE;` should be inside `if (hasInterface) then {`